modules/akka-http/src/main/scala/sup/modules/akkaHttp.scala

```diff
@@ -0,0 +1,61 @@
+package sup.modules
+
+import akka.http.scaladsl.marshalling.ToEntityMarshaller
+import akka.http.scaladsl.model.{StatusCode, StatusCodes}
+import akka.http.scaladsl.server.Directives.{path => akkaPath, _}
+import akka.http.scaladsl.server.{RequestContext, Route}
+import cats.effect.Effect
+import cats.syntax.functor._
+import cats.syntax.reducible._
+import cats.{Functor, Reducible, Semigroup, ~>}
+import sup.{Health, HealthCheck, HealthResult}
+
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+
+object akkaHttp {
+
+  /**
+   * Builds a Route value that'll check the result of the healthcheck, and,
+   * if it's sick, return ServiceUnavailable (Ok otherwise). See [[healthCheckResponse]]
+   * for an alternative that doesn't provide a route matcher.
+   *
+   * The semigroup is used only to determine whether the overall health status is healthy.
+   **/
+  def healthCheckRoutes[F[_] : Effect, H[_] : Reducible](
+    healthCheck: HealthCheck[F, H],
+    path: String = "health-check")
+  (implicit marshaller: ToEntityMarshaller[HealthResult[H]], combineHealthChecks: Semigroup[Health]): Route = {
```

Comment:
I think we can remove these Semigroup parameters, I did this in probably all the rest of the project because it seemed redundant...